internal/pkg/scaffold/helm/dockerfile.go

```diff
@@ -46,4 +46,8 @@ const dockerFileHelmTmpl = `FROM quay.io/operator-framework/helm-operator:{{.Ima
 
 COPY watches.yaml ${HOME}/watches.yaml
 COPY {{.HelmChartsDir}}/ ${HOME}/{{.HelmChartsDir}}/
+
+RUN find ${HOME} -type f -exec chmod -R g+rw {} \; && \
+    find ${HOME} -type d -exec chmod -R g+rwx {} \; && \
+    chmod -R g+rwx /usr/local/bin
```

Comment:
Since this Dockerfile doesn't COPY anything into `/usr/local/bin`, I don't think we need this last `chmod` command here.